meson.build

```diff
@@ -0,0 +1,87 @@
+project(
+  'fvwm3',
+  'c',
+
+  version: '1.0.5',
+  license: 'GPLv2',
+  meson_version: '>=0.60.0',
+  default_options: [
+    'c_std=c11',
+    'warning_level=2',
+    'werror=false',
+    ],
+  )
+
+add_project_arguments(
+  [
+    '-Wno-unused-parameter',
+    '-Wno-missing-braces',
+    '-Wno-unused-result',
+    '-Wvla',
+    '-Wundef',
+    ],
+  language: 'c',
+  )
+
+add_project_arguments('-D_GNU_SOURCE', language : 'c')
+add_project_arguments('-D_POSIX_C_SOURCE=200809L', language: 'c')
+
+cc = meson.get_compiler('c')
+
+datadir = get_option('datadir')
+sysconfdir = get_option('sysconfdir')
+prefix = get_option('prefix')
+
+version = '"@0@"'.format(meson.project_version())
+git = find_program('git', native: true, required: false)
+if git.found()
+  git_commit = run_command([git, 'rev-parse', '--short', 'HEAD'], check: false)
+        git_branch = run_command([git, 'rev-parse', '--abbrev-ref', 'HEAD'], check: false)
+        if git_commit.returncode() == 0 and git_branch.returncode() == 0
+          version = '"@0@-@1@ (" __DATE__ ", branch \'@2@\')"'.format(
+            meson.project_version(),
+            git_commit.stdout().strip(),
+            git_branch.stdout().strip(),
+            )
+        endif
+endif
+add_project_arguments('-DVERSIONINFO=@0@'.format(version), language: 'c')
+
+# Dependencies - REQUIRED
+libevent = dependency('libevent', required: true)
+libx11 = dependency('x11', required: true)
+libxt = dependency('xt', required: true)
+xft = dependency('xft', required: true)
+xrandr = dependency('xrandr', required: true)
+xrender = dependency('xrender', required: true)
+
+# Deoendencies - OPTIONAL
+asciidoctor = find_program(['asciidoctor'], required: false)
+shar = find_program(['shar'], required: false)
+go = find_program(['go'], required: false)
+fontconfig = dependency('fontconfig', required: false)
+freetype = dependency('freetype2', version: '>=2.12.1', required: false)
+fribidi = dependency('fribidi', required: false)
+ncurses = dependency('ncurses', required: false)
+libpng = dependency('libpng', required: false)
+readline = dependency('readline', required: false)
+rsvg = dependency('librsvg-2.0', required: false)
+sm = dependency('sm', required: false)
+xcursor = dependency('xcursor', required: false)
+xext = dependency('xext', required: false)
+xi = dependency('xi', required: false)
+xpm = dependency('xpm', required: false)
+
+
+#fvwm_includes = include_directories('fvwm', 'libs')
+#subdir('fvwm')
+#subdir('modules')
+
+summary({
+'libevent': libevent.found(),
+'libx11': libx11.found(),
+'libxt': libxt.found(),
+'xft': xft.found(),
+'xrandr': xrandr.found(),
+'xrender': xrender.found(),
```

Comment:
You can pass the dependency objects here directly, without .found(), since Meson 0.57.0 (which you require anyway) -- this will also print boolean "YES" or "NO", but includes the version of the dependency that was found.

Reply:
Ditto.